src/javascript/audio.js

```diff
@@ -8,18 +8,26 @@ export default class Audio {
       bufferLength: null,
       dataArray: null,
     };
-    this.maxVolume = 0.5;
+    this.maxVolume = 0.3;
     this.player = null;
+    this.muteBg = false;
     document.addEventListener("visibilitychange", () => {
-      if (this.player) this.mute(document.hidden);
+      if (this.player && !this.muteBg) this.mute(document.hidden);
     });
   }
 
-  async loadSong(songSrc, asBackground, loadedCallback, finishedCallback) {
+  async loadSong(
+    songSrc,
+    asBackground,
+    loadedCallback,
+    finishedCallback,
+    errorCallback
+  ) {
     this.player?.unload();
+    this.asBackground = asBackground;
 
     this.player = new Howl({
-      volume: this.maxVolume,
+      volume: this.muteBg && asBackground ? 0 : this.maxVolume,
```

Comment:
```suggestion
      volume: (this.muteBg && asBackground) ? 0 : this.maxVolume,
```

Reply:
Ahh fork, this must causes at least 1% of the background audio bug.

Reply:
I don't think this is actually a problem? Is just formatting, I'd think the && has higher precedence?